.goreleaser.yaml

```
@@ -0,0 +1,69 @@
+builds:
+  - main: ./cmd/zed
+    id: zed
+    binary: zed
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -X github.com/brimdata/zed/cli.Version={{ .Version }}
+    goarch:
+      - amd64
+      - arm64
+    goos:
+      - linux
+      - windows
+      - darwin
+  - main: ./cmd/zq
+    id: zq
+    binary: zq
+    env:
+      - CGO_ENABLED=0
+    goarch:
+      - amd64
+      - arm64
+    goos:
+      - linux
+      - windows
+      - darwin
+archives:
+  - name_template: zed-{{ .Version }}.{{ .Os }}-{{ .Arch }}
+    format_overrides:
+      - goos: windows
+        format: zip
+    files:
+      - LICENSE*
+      - README*
```

I think we want these two files.
```suggestion
      - LICENSE.txt
      - acknowledgments.txt
```